tests/tools/plugins/null_transform.c

```diff
@@ -1,6 +1,6 @@
 /** @file
```

Comment:
Why is this duplicated?

Reply:
The plugin file is copied over to tests/tools/plugins directory to be built from within the test.